README.md

```diff
@@ -51,36 +49,43 @@ var nav = [
   {id: 'about', label: 'About', path: '/about'},
   {id: 'discover', label: 'Discover', path: '/discover'},
 ]
+
 <SlideMenu
   active={this.state.slideMenuActive}
   nav={nav}
   reactRouter={true}
   closeMenu={() => this.setState({slideMenuActive: false})}>
-  <YOUR APP />
-</SlideMenu
+  <YOUR_APP />
```

Comment:
This just made the highlighting in my editor more consistent, maybe `YourApp` would be a better choice 🤷‍♂